server/server.js

```diff
@@ -18,19 +18,21 @@ app.use(cors())
 dotenv.config({ path: join(__dirname, "./config/config.env") });
 
 
-app.get("/topArabicSongs", scrapeTopArabicSongs)
-app.get("/topHebrewSongs", scrapeTopHebrewSongs)
+app.use("/api/v1/", scrappingRoutes)
+
 
 const PORT = process.env.PORT || 5000;
 const NODE_ENV = process.env.NODE_ENV
 
 let server;
-connectDB().then(() => {
+
+connectDB()
+
+.then(() => {
```

Comment:
this is still connected. you say you want app.listen to happen after connectDB. they should not be connected at all.